src/index.html

```diff
@@ -1,13 +1,96 @@
+<!-- #region head -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>BOSE</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap" rel="stylesheet">
+    <link rel="icon" href="./images/logo.png"
+          type="image/x-icon">
+    <link rel="stylesheet" href="styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <body class="page__body">
+    <header class="header">
+      <div class="top-bar">
+        <a href="#" class="top-bar__logo">
+          <img src="./images/logo.png" alt="bose logo">
+        </a>
+        <div class="top-bar__icons">
+          <a href="tel:+1 234 5555-55-55" class="icon icon--phone"></a>
+          <a href="#menu" class="icon icon--menu"></a>
+        </div>
+      </div>
+      <div class="header__bottom">
+        <div class="header__waves"></div>
+        <h1 class="header__title">The world shades. <br>
+        Your music shines.
+        </h1>
+      </div>
+    </header>
+
+    <aside class="menu page__menu" id="menu">
+      <div class="top-bar menu__top">
+        <a href="#" class="top-bar__logo">
+          <img src="./images/logo.png" alt="bose logo">
+        </a>
+        <a href="#" class="icon icon--close"></a>
+      </div>
+      <div class="menu__content">
+        <nav class="nav menu__nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a class="nav__link" href="#">Home</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#recomended">Recomended</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#categories">Categories</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#how-to-buy">How to buy</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#contacts">Contacts</a>
+            </li>
+          </ul>
+        </nav>
+        <div class="call">
+          <a href="tel:+1 234 5555-55-55" class="menu__phone-number">+1 234 5555-55-55</a>
+          <a  class="menu__call-to-order">Call to order</a>
+        </div>
```

Comment:
There is also a BEM violation here - the block contains elements of another block, so either rewrite them or make `.call' not a block but an element